packages/deep-merge/tsconfig.eslint.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.json",
+  "exclude": []
```

Comment:
I don't know why we keep adding this `"exclude"` property if we are setting it to an empty array. Am I missing something? 😅